### PR TITLE
 Updated Webpage of Women helpline numbers in Site under the file helpline_numbers.

### PR DESCRIPTION
**Pull request by mansijain980**
##Updated with new Features

Fixes:  #124
#### Description 

I have added a lot of features on webpage of women helpline by bootstrap , css , html , javascipt  likeuser can go above the page directly by clicking button .
Statewise helpline numbers are also linked . 
An attractive page. 
Now , it is easy for user to search for a particular sate helpline numbers .

## Type of change
- [ ] New feature (non-breaking change which adds functionality)
- 

## Checklist:
-  My code follows the code style of this project.
-  My code follows the style guidelines of this project.
-  I have performed a self-review of my own code.

## Screenshots

 
image

### CODE REVIEW

**Comment by mansijain980** [comment]
Hey ! @sakshikhachane  I have Requested a PR for this issue . Please let me know if there is any problem with something that i have done .
Please marge a pull request . 
Thanks

**Comment by swapnilsparsh** [comment]
@sakshikhachane Have a look.

File: main_app/static/css/styles.css [hunk 1]
```diff
@@ -1,18 +1,116 @@
+@import url("https://fonts.googleapis.com/css?family=Sunflower:300");
```
Comment (sakshikhachane) [inline main_app/static/css/styles.css:1]: For now please use inline CSS only. 
Don't add CSS in `styles.css` file.

File: main_app/templates/main_app/login.html [hunk 2]
```diff
@@ -1,6 +1,4 @@
-<!DOCTYPE html>
-{% extends "main_app/common.html" %}
-{% load static %}
+<!DOCTYPE html> {% extends "main_app/common.html" %} {% load static %}
```
Comment (sakshikhachane) [inline main_app/templates/main_app/login.html:1]: Please remove the `login.html` file from the PR.

File: main_app/templates/main_app/register.html [hunk 3]
```diff
@@ -1,90 +1,45 @@
...
-                        <br>
-                        <div style="display: flex; align-items: center;
-                    justify-content: center;">
-                            <strong>Already on RESCUE? </strong><a href="/login" style="color: rgb(30,142,62);"><b
-                                    class="sign-in"><u>Sign
-                                        In</u></b></a>
-                        </div>
-                    </form>
-
+<html lang="en">
```
Comment (sakshikhachane) [inline main_app/templates/main_app/register.html:2]: Remove `register.html` file from PR.

**Review by sakshikhachane — CHANGES_REQUESTED** [review]
@mansijain980
- [ ]  Please keep the format/ design of the `helpline_numbers` page the same as previous.
- [ ] Just add the rest of the numbers below it. 
- [ ] Also, keep your branch even with the `master` branch of this repo.

**Comment by sakshikhachane** [comment]
@mansijain980 Any updates?

**Comment by mansijain980** [comment]
> @mansijain980 Any updates?

Yes maam ... i m working on it . Actually i started working on some other issue. By tomorrow evening , I will commit changes and append rows on that table .
Thanks.

**Comment by mansijain980** [comment]
@sakshikhachane ma'am can u tell me the color of background? It should be white na?

**Comment by sakshikhachane** [comment]
> @sakshikhachane ma'am can u tell me the color of background? It should be white na?

Keep the background the same as it was before doing the changes.

**Comment by mansijain980** [comment]
@sakshikhachane Can u check it now please?

**Review by sakshikhachane — CHANGES_REQUESTED** [review]
@mansijain980 

- [ ] You have to remove the files from PR, not from your forked repo. Please revert the changes or make a new PR.

- [ ] Please fix this - 
image

- [ ] Remove this as well. 
image

**Pull request closed without merging**